lib/David_tic_tac_toe.rb

```diff
@@ -5,24 +5,24 @@
 require_relative 'David_tic_tac_toe/player'
 require_relative 'David_tic_tac_toe/game'
 module DavidTicTacToe
-  @game = DavidTicTacToe::Game.new(DavidTicTacToe::Board.new, DavidTicTacToe::Player.new('David', 'X'),
```

Comment:
This raised scooping problems because it was tied to the class/module, not the calling instance.